client/extensions/woocommerce/woocommerce-services/views/label-settings/label-settings.js

```diff
@@ -200,6 +201,47 @@ class ShippingLabels extends Component {
 		);
 	};
 
+	renderEmailReceiptsSection = () => {
+		const {
+			emailReceipts,
+			translate,
+			masterUserName,
+			masterUserLogin,
+			canEditSettings,
+			canEditPayments,
+			setFormDataValue,
+		} = this.props;
+
+		if ( ! isBoolean( emailReceipts ) ) {
+			return null;
+		}
+
+		const onChange = () => setFormDataValue( 'email_receipts', ! emailReceipts );
+
+		return (
+			<FormFieldSet>
+				<FormLabel className="label-settings__cards-label">
+					{ translate( 'Email Receipts' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **36** times:
`translate( 'Email Receipt' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).